rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -3308,6 +3308,200 @@ public void saveNextPegoutHeight_after_RSKIP271() {
         );
     }
 
+    @Test
+    public void getPegoutCreationEntry_before_RSKIP298() {
+        Repository repository = mock(Repository.class);
+        BridgeStorageProvider provider = new BridgeStorageProvider(
+            repository, PrecompiledContracts.BRIDGE_ADDR,
+            config.getNetworkConstants().getBridgeConstants(), activationsBeforeFork
+        );
+
+        Sha256Hash hash = Sha256Hash.ZERO_HASH;
+
+        DataWord storageKeyForPegoutCreationIndex = DataWord.fromLongString("pegoutCreationIndex-" + hash);
+        when(
+            repository.getStorageBytes(
+                PrecompiledContracts.BRIDGE_ADDR,
+                storageKeyForPegoutCreationIndex
+            )
+        ).thenReturn(BridgeSerializationUtils.serializeKeccak256(Keccak256.ZERO_HASH));
```

Comment:
Why not? Shouldn't both pre and post-fork tests?

Reply:
Sorry, I didn't quite select the right lines. The test is needed yes, I meant that what's not needed is the whole repository setup since the method will return empty immediately.

But, seeing the assertion you make below then it does make sense :)